exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/main_config.ipynb

```diff
@@ -7,9 +7,16 @@
    "source": [
```

Comment:
<blockquote>Here we are going to set or update one lot of configuration parameters or the other depending on the choice of the database. </blockquote><p>This sentence sounds a bit confusing for me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
yep, sounds good.

Small typo in:

The set of configuration parameters depend**s** on the choice of the database.